LICENSE

```diff
@@ -1,2 +1,996 @@
-Copyright (c) 2016 Massachusetts Institute of Technology
-All rights reserved.
+Copyright (c) 2015-2016 Massachusetts Institute of Technology
+
+This library is free software; you can redistribute it and/or modify it under
+the terms of the GNU Lesser General Public License as published by the Free
+Software Foundation; either version 2.1 of the License, or (at your option) any
+later version.
+
+This library is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with this library (copied below); if not, write to the Free Software Foundation,
+Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA.
+
+Other licensing options may be available if required; contact the MIT Technology
```

Comment:
I don't think that we can do this. I'm going to want to run this by TLO before we make such an open ended statement. I think we have permission to release it under LGPL 2.1, but no other license. I'm not very clear on the legality of these things. I might make this statement a bit weaker. Also, I don't think TLO necessarily wants us to list them as a contact for this.